src/transformers/models/whisper/modeling_whisper.py

```diff
@@ -801,7 +799,7 @@ def forward(
 
 # Copied from transformers.models.mbart.modeling_mbart.MBartDecoderLayer with MBart->Whisper, MBART->WHISPER
```

Comment:
I'll propagate changes to all other MBart derived modules when we're happy with the design